deepchem/trans/tests/test_transformers.py

```diff
@@ -482,3 +485,42 @@ def test_featurization_transformer(self):
 
     self.assertEqual(new_train.y.shape, train.y.shape)
     self.assertEqual(new_train.X.shape[-1], fp_size)
+
+  def test_data_transformers(self):
+    mnist = input_data.read_data_sets("MNIST_data/", one_hot=True)
+    # extracting validation set of MNIST for testing the DataTransforms
+    valid = dc.data.NumpyDataset(mnist.validation.images,
+                                 mnist.validation.labels)
+    # extract only the images (no need of the labels)
+    data = (valid.X)[0]
```

Comment:
what is this? How is this different than valid.X?  Is there some syntax I don't understand here?

Reply:
So `valid.X` has 10000 images and I just want one of those images.

Reply:
SGTM

Reply:
test functions must begin with test_